app/routes/members-area.my-account/route.module.scss

```diff
@@ -0,0 +1,4 @@
+.page {
+    text-align: center;
+    font: var(--heading5);
```

Comment:
Are you sure about `--heading5` as the body text?

Reply:
it's temporary for the "This page is under construction" message and going to removed when the actual page is implemented. But I can wrap the message in another div and apply the style there, for this to make more sense.